src/DynamoCoreWpf/Views/Core/NodeView.xaml.cs

```diff
@@ -601,8 +604,16 @@ private void OnNodeViewMouseLeave(object sender, MouseEventArgs e)
             ViewModel.ZIndex = oldZIndex;
 
             //hide the node autocomplete marker if mouse leaves the node
-           TryHideAutoCompleteMaker();
-            
+            if (IsAutoCompleteMarkerDisabled()) return;
+            if (Mouse.Captured is DragCanvas)
+            {
+                Dispatcher.BeginInvoke(new Action(TryHideAutoCompleteMaker), DispatcherPriority.Loaded);
+            }
+            else
+            {
+                Dispatcher.DelayInvoke(autoCompleteMarkerDelay, TryHideAutoCompleteMaker);
```

Comment:
Do you think the delay would be something we can define on XML layer? @johnpierson @BogdanZavu Just curious

Reply:
I am not too sure but can research. Would be keen to your thoughts @BogdanZavu

Reply:
I was worried that if multiple nodes have focus it will try to show the markers for all of those nodes, but maybe that is ok?

Reply:
Yeah, imo I think it ok to show the marker for what's in the selection. I think we're inline with other controls that appear on selection.

Reply:
I think that makes sense to me. So effectively tie it to the blue selection border almost?

Reply:
I think we should tie it to the focus event if possible. The blue border control behavior might change in the future.